src/rect.rs

```diff
@@ -615,4 +656,38 @@ mod tests {
         );
     }
     */
+
+    #[test]
+    fn fix_aspect_ratio() {
+        use std::f64;
```

Comment:
The Travis CI here uses a rather old rust compiler version. I think a better approach would be to improve the CI situation. I'll see if I can cook something up.

Reply:
I'm happy to do this as an alternative, what are others' thoughts?

Reply:
@raphlinus was it an intentional decision to support older rust compilers? I seem to recall some conversation to that effect but I might be confused.

Reply:
I think not so much an intentional decision to support older compilers, but so we know when to explicitly raise the MSRV. I'm fine with requiring latest stable actually.

At some point in the future (when people want to distribute druid apps on Debian, for example), we might want to have a more conservative MSRV policy.